packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -71,7 +71,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         {...{ onBlurCapture: this._hideTooltip }}
         onMouseEnter={this._onTooltipMouseEnter}
         onMouseLeave={this._onTooltipMouseLeave}
-        aria-describedby={setAriaDescribedBy && isTooltipVisible && content ? tooltipId : undefined}
+        aria-describedby={setAriaDescribedBy && isTooltipVisible && isContentPresent ? tooltipId : undefined}
```

Comment:
When you have multiple conditionals hooked up to a ternary value, you should use parenthesis to ensure that your conditions are evaluated as expected. It's a good practice to be sure of ordering and readability. For example:

Instead of:
```tsx
setAriaDescribedBy && isTooltipVisible && isContentPresent ? tooltipId : undefined
```

Use this:
```tsx
(setAriaDescribedBy && isTooltipVisible && isContentPresent) ? tooltipId : undefined